app/models/attribute/GlobalAttributeTable.scala

```diff
@@ -131,18 +131,28 @@ object GlobalAttributeTable {
     globalAttributes.list
   }
 
+  def toInt(s: String): Option[Int] = {
```

Comment:
Could this be updated to take in an `Option[String]` so that in the rest of the code you can just do `toInt(severity)` instead of `toInt(severity.getOrElse("6"))`?